### DISCUSSION TITLE
Machine Learning Models

### DISCUSSION THREAD

**Discussion by anubhavagr** [OpInCo-Community/HacktoberWeek#31] [Ideas]
- [ ] - Listing out the models that is available at tensorflowhub, huggingface, pytorchhub
- [ ] - How to implement it
- [ ] - New applications of these trained models

**Comment by Raporaz03** [comment]
how to fix error   **Image Is Truncated**

**Reply by anubhavagr** [reply]
Kindly share the screenshot of the error

**Reply by Raporaz03** [reply]
solved
